lib/dynflow/active_job/queue_adapter.rb

```diff
@@ -28,6 +28,10 @@ def plan(attributes)
         def run
           input[:job_class].constantize.perform_now(*input[:job_arguments])
         end
+
+        def label
+          input[:job_class]
```

Comment:
I think it'd be nice to have `job_class` or `input[:job_class].constantize.label`. The reason is that if one has 2 jobs scheduled, for example SyncLDAPUsers(usergroup), I want to check on initialization if there's a job scheduled for that already. 

```
Dynflow.world.find_execution_plans :label => "SyncLDAPUser" 
```

is not good enough, as it assumes only 1 job will be scheduled of the same class at any point in time.

```
Dynflow.world.find_execution_plans :label => "sync-ldap-user-myfavoritegroup" 
```

is different and does not make that assumption (it allows me to iterate over all usergroups and only schedule syncs for the ones that are not scheduled). 

It'd be as simple as defining the label in the ActiveJob as 

```
def label
    "sync-ldap-users-#{@usergroup}"
end
```